tools/lint.sh

```diff
@@ -8,6 +8,9 @@ if [[ ! -z "$1" ]]; then
     cd "$1"
 fi
 
+# Do an initial pass with --all-targets
+cargo clippy --all --all-features --all-targets
```

Comment:
This feels like a narration comment re-iterating _what_ but not addressing _why_
Why is there an initial pass with `--all-targets` instead of including `--all-targets` in the loop for each workspace?